pkg/cli/cvd.go

```diff
@@ -159,6 +162,50 @@ func (c *cvdCreator) Create() ([]*hoapi.CVD, error) {
 	return c.createCVDFromAndroidCI()
 }
 
+const uaEnvConfigTmpl = `
+{
+  "common": {
+    "host_package": "@user_artifacts/{{.ArtifactsDir}}"
+  },
+  "instances": [
+    {
+      "vm": {
+        "memory_mb": 8192,
+        "setupwizard_mode": "OPTIONAL",
+        "cpus": 8
+      },
+      "disk": {
+        "default_build": "@user_artifacts/{{.ArtifactsDir}}"
+      },
+      "streaming": {
+        "device_id": "cvd-1"
+      }
+    }
+  ]
+}
+`
+
+type uaEnvConfigTmplData struct {
+	ArtifactsDir string
+}
+
+func (d *uaEnvConfigTmplData) Value() map[string]interface{} {
+	tmpl, err := template.New("").Parse(uaEnvConfigTmpl)
+	if err != nil {
+		panic(err)
+	}
+	var b bytes.Buffer
+	err = tmpl.Execute(&b, uaEnvConfigTmplData{ArtifactsDir: d.ArtifactsDir})
+	if err != nil {
+		panic(err)
+	}
+	result := make(map[string]interface{})
+	if err := json.Unmarshal(b.Bytes(), &result); err != nil {
+		panic(err)
+	}
```

Comment:
Uniting the style for error handling? Former reuses `err` variable, but latter declares `err` variable inside conditional statement.